src/util/api.js

```diff
@@ -387,11 +461,16 @@ export const apiGetPlaySession = ({ widgetId }) => {
 		})
 }
 
-export const apiGetQuestionSet = (instId, playId) => {
+export const apiGetQuestionSet = (instId, playId = null) => {
 	return fetch('/api/json/question_set_get/', fetchOptions({ body: `data=${formatFetchBody([instId, playId])}` }))
 		.then(qset => qset.json())
 }
 
+export const apiGetQuestionSetHistory = (instId) => {
+	return fetch('/api/instance/history', fetchOptions({ body: `data=${formatFetchBody([instId])}` }))
```

Comment:
I can understand the confusion with the API calls, but this is using the formatting required for the old (v1) API: those hitting `/api/json/xxx`. Newer API endpoints are added within the requisite controllers, in this case `controllers/api/instance.php`. This particular request is formatted for v1, which requires everything to be POST - but the instance API controller is expecting this particular call to be a GET request. It currently 405s.